src/blueprint/mustache.rs

```diff
@@ -101,14 +101,14 @@ impl<'a> MustachePartsValidator<'a> {
 }
 
 impl FieldDefinition {
-    pub fn validate_field(&self, type_of: &config::Type, config: &Config) -> Valid<(), String> {
+    pub fn validate_field(self, type_of: &config::Type, config: &Config) -> Valid<Self, String> {
```

Comment:
Why are we making this change?

Reply:
Or open another PR to it

Reply:
It takes a reference and validates. That's a good design for a validator. Let's revert this please.

Reply:
Reverted.